### ISSUE TITLE
When double-clicking to select text in the query editor, the period should be treated as a word separator

### ISSUE THREAD

**Issue by samuelbradshaw** [Sequel-Ace/Sequel-Ace#377]
Given a sample query like this:

```
SELECT Songs.id, Song.title, Collections.title
FROM Songs
JOIN Collections ON Songs.collectionId = Collections.id
```

**Ideal behavior:** Double-clicking `title` from `Song.title` only selects `title`. This is how, for example, Xcode behaves, recognizing that the period is usually a separator between words in code.

**Current behavior:** Double-clicking `title` from `Song.title` selects the full identifier `Song.title`.

**Comment by Kaspik** [comment]
Implemented for 3.0.0 in https://github.com/Sequel-Ace/Sequel-Ace/pull/591

**Comment by Kaspik** [comment]
Has to be reverted due to performance issues when subclassing NSTextStorage - https://github.com/Sequel-Ace/Sequel-Ace/pull/715

Related: https://www.thecope.net/2019/09/15/resolving-slow-performance.html

I think I'll leave this closed because it's not easily achievable right now.
